xml/dm_registers.xml

```diff
@@ -474,8 +474,10 @@
            permission checks that apply based on the current architectural
            state of the hart performing the access, or with a relaxed set of
            permission checks (e.g. PMP restrictions are ignored).  The
-           details of the latter are implementation-specific.  When set to 0,
-           full permissions apply; when set to 1, relaxed permissions apply.
+           details of the latter are implementation-specific.
+
+           When set to 0, full permission checks apply. When set to 1,
+           relaxed permission checks apply.
```

Comment:
```suggestion
           <value v="0" name="full_checks">
           Full permission checks apply.
           </value>
           
           <value v="1" name="relaxed_checks">
           Relaxed permission checks apply.
           </value>
```

Reply:
Done with a minor change: Underscores replaced by spaces in the `name` attribute.